### PR TITLE
Fix annotation value read :bug:

### PR DESCRIPTION
**Pull request by Freymaurer**
Annotation values are now always written to string instead of any of with number option. This change was not correctly adapted in the read for annotationValue.

### CODE REVIEW

**Comment by Freymaurer** [comment]
I will leave the PR up but release from this bug fix branch to work on ARCitect, Swate development.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+### 1.0.4+167fae8 (Released 2024-1-15)
```
Comment (HLWeil) [inline RELEASE_NOTES.md:1]: Commits for 1.0.2 and 1.0.3 got lost (by me)? 

Did you increase patch version three times here?

**Review by HLWeil — APPROVED** [review]
lgtm, fixed the problems in ARCitect?

File: RELEASE_NOTES.md (hunk 1 above)
Comment (Freymaurer) [reply RELEASE_NOTES.md:1]: 1.0.2, 1.0.3 and 1.0.4 are all part of this PR and are correctly written to RELEASE_NOTES.md

**Pull request merged**